backend/src/database/sql_helpers.rs

```diff
@@ -0,0 +1,66 @@
+use std::time::Duration;
+
+use sqlx::{
+    postgres::{PgConnectOptions, PgPoolOptions, PgRow},
+    PgPool, Postgres,
+};
+use tokio::sync::OnceCell;
+
+use crate::{general_helpers::ENV_VARS, socket_handlers::types::AppError};
+
+use super::types::{comment::Comment, image::Image, post::Post, user::User};
+
+async fn table_exists_with_columns(table: &str, columns: &[&str]) -> Result<bool, sqlx::Error>
+where
+{
+    let pool = get_pool().await;
+    let table_exists_query = format!("SELECT EXISTS (SELECT * FROM {});", table);
+
+    let table_exists = match sqlx::query::<_>(&table_exists_query).fetch_one(&pool).await {
+        Ok(_) => Ok(true),
+        Err(e) => {
+            if e.to_string().contains("does not exist") {
+                Ok(false)
+            } else {
+                Err(e)
+            }
+        }
+    }?;
+
+    if !table_exists {
+        return Ok(false);
+    }
+
+    Ok(true)
```

Comment:
kan du ikke bare return match sql::query::<_>..... I stedet for at matche længere nede og så returne